lte/cloud/go/services/subscriberdb/servicers/subscriberdb_servicer.go

```diff
@@ -207,6 +207,25 @@ func (s *subscriberdbServicer) ListSubscribers(ctx context.Context, req *lte_pro
 	return listRes, nil
 }
 
+func (s *subscriberdbServicer) ListSuciProfiles(ctx context.Context, req *lte_protos.ListSuciProfilesRequest) (*lte_protos.ListSuciProfilesResponse, error) {
+	gateway := protos.GetClientGateway(ctx)
+	if gateway == nil {
+		return nil, status.Errorf(codes.PermissionDenied, "missing gateway identity")
+	}
+	if !gateway.Registered() {
+		return nil, status.Errorf(codes.PermissionDenied, "gateway is not registered")
+	}
+	networkID := gateway.NetworkId
+
+	var suciProtos []*lte_protos.SuciProfile
+	suciProtos, _ = subscriberdb.LoadSuciProtosPage(ctx, networkID)
```

Comment:
General: don't drop errs -- either bubble them up (with wrapper), or, under certain circumstances, log them. In this case, we want to bubble

Reply:
got it